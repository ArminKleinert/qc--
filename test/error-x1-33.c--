register bits1 one;

foo() {
    bits32 x, y;
    bits16 a ,b;

    x, y = a, b;
}
