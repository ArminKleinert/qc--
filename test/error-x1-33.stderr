Character 77-89 Error: type mismatch in assignment
